test/test_class1_affinity_predictor.py

```diff
@@ -13,7 +13,6 @@
 
 from mhcflurry import Class1AffinityPredictor
 
-from nose.tools import eq_, assert_raises
```

Comment:
The CI is failing because you've removed `assert_raises` here, yet the function is still being called in this test file.